Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -134,6 +137,19 @@ final class PurchasesOrchestrator {
         }
 
         Task {
+            #if os(iOS) || targetEnvironment(macCatalyst) || VISION_OS
+            #if swift(>=5.8)
+            if #available(iOS 16.4, *) {
```

Comment:
Ditto so we don't forget
```suggestion
            if #available(iOS 16.0, *) {
```